storage_backend_sftp/components/sftp_adapter.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2017 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# Copyright 2019 Camptocamp SA (http://www.camptocamp.com).
+# Copyright 2020 ACSONE SA/NV (<http://acsone.eu>)
+# @author Simone Orsi <simahawk@gmail.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+import errno
+import logging
+import os
+from contextlib import contextmanager
+from io import StringIO
+
+from odoo.addons.component.core import Component
+
+_logger = logging.getLogger(__name__)
+
+try:
+    import paramiko
+except ImportError as err:  # pragma: no cover
+    _logger.debug(err)
```

Comment:
```suggestion
import paramiko
```

Since it's already in the module's `external_dependencies`, there's no need to wrap this import